sql/ifdb-to-ifarchive-scrubber.sql

```diff
@@ -154,3 +151,7 @@ drop table gametimes_mv;
 drop trigger playertime_insert;
 drop trigger playertime_update;
 drop trigger playertime_delete;
+
+-- delete users with acctstatus D (pending activation) R (pending review) B (banned)
+delete from users where acctstatus != 'A';
```

Comment:
If  you mean D/R/B and expect other statuses in the future, then maybe it's better to use `IN ('D', 'R', 'B')`